synedrion/src/paillier/ring_pedersen.rs

```diff
@@ -1,134 +1,166 @@
+/// Implements the Definition 3.3 from the CGGMP'21 paper and related operations.
```

Comment:
We should be careful about references now that there are two versions of the paper. How about we use `CGGMP21` for the original version and `CGGMP21/24` for the new version?

Which one is this?

Reply:
I am careful, this particular definition is 3.3 in both :) But there will be updates once I start with ZK proofs and protocols.